src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +45,9 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream()
+                .filter(number -> number > 3)
+                .mapToInt(number -> number * 2)
+                .sum();
```

Comment:
```suggestion
                .map(number -> number * 2)
                .reduce(0, Integer::sum);
```
혹은
```suggestion
                .reduce(0, (sum, number) -> sum + number * 2);
```
이렇게 표현할 수 있습니다!